manifests/o/OpenTTD/OpenTTD/12.1/OpenTTD.OpenTTD.locale.en-US.yaml

```
@@ -0,0 +1,27 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: OpenTTD.OpenTTD
+PackageVersion: 12.1
+PackageLocale: en-US
+Publisher: OpenTTD
+PublisherUrl: https://www.openttd.org/
+PublisherSupportUrl: https://www.openttd.org/contact.html
+PrivacyUrl: https://www.openttd.org/policy.html
+PackageName: OpenTTD
+PackageUrl: https://www.openttd.org/
+License: GPL-2.0
+LicenseUrl: https://github.com/OpenTTD/OpenTTD/blob/master/COPYING.md
+ShortDescription: OpenTTD is a transport simulation game based upon the popular game Transport Tycoon Deluxe, written by Chris Sawyer. It attempts to mimic the original game as closely as possible while extending it with new features.
+Moniker: openttd
+Tags:
+- game
+- ttd
+- open source game
+- trnsport tycoon
+- tycoon

```

Standardize Tag Format
```suggestion
Tags:
- game
- ttd
- open-source-game
- transport-tycoon
- tycoon
```